port_exposer/src/main.rs

```diff
@@ -1,8 +1,9 @@
 // Copyright (c) 2017-2019, Substratum LLC (https://substratum.net) and/or its affiliates. All rights reserved.
-pub mod port_exposer;
 
 use crate::port_exposer::PortExposer;
 
+pub mod port_exposer;
```

Comment:
Convention is to put `mod` statements before `use` statements. Please move the `pub mod port_exposer` line back above the `use crate::port_exposer::PortExposer` line.